docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: "3"
+
+services:
+  # Servers
+  appserver:
+    image: tu2bo-appserver_app
+    build:
+      context: ./tu2bo-appserver/app
+      dockerfile: Dockerfile.web
+    environment: 
+      - PORT=5000
+      - AUTH_URI=http://authserver:3000
+      - TUTUBO_ENV=PROD
+    ports:
+      - "5000:5000"
+    networks:
+      - default
+      - appserver-network
+      - system-network
+  
+  authserver:
+    image: tu2bo-authserver_app
+    build:
+      context: ./tu2bo-authserver/app
+      dockerfile: Dockerfile.web
+    environment: 
+      - PORT=3000
+    ports:
+      - "3000:3000"
+    networks:
+      - authserver-network
+      - system-network
+
+  # DBs
+  appserver-db:
+    image: mongo:3.6.4
+    networks:
+      - appserver-network
+
+  authserver-db:
+    image: mongo:3.6.4
+    networks:
+      - authserver-network
+
+networks:
```

Comment:
What's the purpose of this?

Reply:
Network are used to "encapsulate" the communication between containers. Only containers that share a network are able to communicate with each other (they are represented as the dotted-line rectangles in the diagram). So this can help us simulate better the fact that we have different services deployed somewhere, and guarantee that they will only talk to who we want them to talk.

Reply:
There also seems to be a way to allow just one network to be accessible for requests to `localhost` from the user's terminal... but I couldn't find how to do that xd

Reply:
Oh sure, fach before hoes

Reply:
But that would have been cool, to only be able to hit the AppServer endpoints, pretending to be the App who is requesting!